lib/normalizer.c

```diff
@@ -849,7 +849,78 @@ grn_inline static const unsigned char *
 grn_nfkc_normalize_unify_alphabet_diacritical_mark(
   const unsigned char *utf8_char, unsigned char *unified)
 {
-  // TODO: Implement function to return `unified` instead of `utf8_char`.
+  if (utf8_char[0] == 0xC3) {
+    /* U+00C0 LATIN CAPITAL LETTER A WITH GRAV À ..
```

Comment:
You don't need to add `À` here to avoid non ASCII characters in source code.
We can know the character only by `U+XXXX CHARACTER NAME`.